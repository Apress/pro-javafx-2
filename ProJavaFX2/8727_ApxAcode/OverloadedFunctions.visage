/*
 * @test
 * @run
 */

function f(i:Integer):Void {
  println("Integer version of f is called with i = {i}.");
}

function f(n:Number):Void {
  println("Number version of f is called with n = {n}.");
}

function f(str:String):Void {
  println("String version of f is called with str = {str}.");
}

f(1024);
f(3.14);
f("Hello, World");
