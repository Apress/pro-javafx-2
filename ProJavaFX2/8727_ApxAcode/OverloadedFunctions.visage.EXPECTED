Integer version of f is called with i = 1024.
Number version of f is called with n = 3.14.
String version of f is called with str = Hello, World.
